__test__/pet-store.spec.ts

```diff
@@ -212,6 +212,33 @@ describe('Pet Store', () => {
         })
       }).toThrow(RequestValidationError)
     })
+
+    test('It returns defined body content type', () => {
+      expect(chowchow.getDefinedBodyContentType('/pets', 'post'))
+        .toMatchInlineSnapshot(`
```

Comment:
Out of interest, why use `toMatchInlineSnapshot` ?

Reply:
I was thinking the snapshot was too small to have a new file by using `toMatchSnapshot`, but I am happy to switch to that.